### ISSUE TITLE
Textfield labels are not easy to read

### ISSUE THREAD

**Issue by dsebastien** [google/material-design-lite#4093]
I've got this feedback from multiple users of an app I've created using MDL. Honestly I also thought the same but thought that it was probably my screen's configuration.

This has pushed me to override the default opacity from 0.12 to 0.50 for `.mdl-textfield__label`.

Is this something that was already mentioned by other users?

**Comment by rj33** [comment]
Yes, I find them hard to read with the default opacity.

Google's specs in this area have evolved over time I think (my default seems higher than 0.12 though at 0.26). The current spec suggests 0.38 for hints:
http://www.google.com/design/spec/style/color.html#color-text-background-colors

The trouble with the text field labels is that they are being interpreted as hints which have the same visibility as disabled items in the spec. MD text field labels are quite like placeholder hints until they are expanded out when a value is entered, so it is debatable if they should be treated like normal texts or labels when acting in a placeholder manner. In any case matching the current spec with 0.38 would be a slight improvement, but I think clarification should be sought from the MD spec people to check if text field labels are considered hints for this purpose, as I have a feeling the text field screenshots in the spec are using something with much higher opacity than what MDL is using right now.

**Comment by surma** [comment]
Yup, you seem to be right that the opacity of the labels has been increased since. We’ll need to adjust here. Thanks for the issue :)
